### ISSUE TITLE
Hide OSTeammate pathways?

### ISSUE THREAD

**Issue by stevejburge** [OSTraining/OSCampus#223]
Maybe have a small pull-down to choose between Owner / system?

![screen shot 2016-03-25 at 1 13 32 pm](https://cloud.githubusercontent.com/assets/3868028/14049404/75b7e046-f28b-11e5-85c1-7980ab3158de.png)

**Comment by billtomczak** [comment]
Yeah - managing pathways in the backend has been much on my mind these days. I like the dropdown idea!
